chia/types/spend_bundle.py

```diff
@@ -59,6 +60,10 @@ def fees(self) -> int:
 
         return amount_in - amount_out
 
+    @property
+    def id(self) -> SpendBundleID:
```

Comment:
Take a look at this line that adds a new `SpendBundleID` property to `SpendBundle`

Reply:
`SpendBundle.name()` has many many uses in many many files.

Reply:
generally I think it's a mistake to add properties that hare costly to compute. We (used to) compute header hashes and coin ids unnecessarily, without the code making it look like it, because of this.
I think it's reasonable to make it visible at the call-site that computing the spend bundle ID is not free, but is in fact a function call.

Reply:
Ya, I can reverse this property, and make it a method call.